near/near-wallet
# Linkdrop and 2FA account creation

Test instructions:
1. Generate neardrop link using: https://near-drop.onrender.com/
2. Change domain to https://near-wallet-pr-700.onrender.com
3. Example link (note that this link does not work): https://near-wallet-pr-700.onrender.com/create/testnet/3gdXfK4fy44tA91xHEhvWaAyPnNqAvRFfq6W5RnbJn647N3QbNnyd7CFZRq8LHy4aVtvAETU9Xht84uep5rxLR6N
